src/pages/placement-groups/PlacementGroupList.tsx

```diff
@@ -0,0 +1,223 @@
+                  placementGroup={placementGroup}
+                  project={projectName}
+                  key="delete"
+                />,
+              ]}
+            />
+          ),
+          role: "cell",
+          "aria-label": "Actions",
+          className: "u-align--right",
+        },
+      ],
+      sortData: {
+        name: placementGroup.name.toLowerCase(),
+        description: placementGroup.description.toLowerCase(),
+        policy: placementGroup.config.policy,
+        rigor: placementGroup.config.rigor,
+        used_by: placementGroup.used_by.length,
+      },
+    };
+  });
+
+  const { rows: sortedRows, updateSort } = useSortTableData({ rows });
+
+  if (isLoading) {
+    return <Spinner className="u-loader" text="Loading..." isMainComponent />;
+  }
+
+  return (
+    <>
+      <CustomLayout
+        mainClassName="placement-groups-list"
+        header={
+          <PageHeader>
+            <PageHeader.Left>
+              <PageHeader.Title>
+                <HelpLink
+                  docPath="/howto/cluster_placement_groups/"
+                  title="Learn how to use placement groups"
+                >
+                  Placement groups
+                </HelpLink>
+              </PageHeader.Title>
+            </PageHeader.Left>
+            <PageHeader.BaseActions>
+              {placementGroups.length > 0 && (
+                <CreatePlacementGroupBtn className="u-no-margin--bottom u-float-right" />
+              )}
+            </PageHeader.BaseActions>
+          </PageHeader>
+        }
+      >
+        <NotificationRow />
+        <Row className="no-grid-gap">
+          <Col size={12}>
+            {placementGroups.length === 0 && (
+              <EmptyState
+                className="empty-state"
+                image={<Icon name="repository" className="empty-state-icon" />}
+                title="No placement groups found"
+              >
+                <p>There are no placement groups in this project.</p>
+                <p>
+                  <DocLink
+                    docPath="/howto/cluster_placement_groups/"
+                    hasExternalIcon
+                  >
+                    Learn how to use placement groups
+                  </DocLink>
+                </p>
+                <CreatePlacementGroupBtn />
+              </EmptyState>
+            )}
+            {placementGroups.length > 0 && (
+              <ScrollableTable
+                dependencies={[placementGroups, notify.notification]}
+                tableId="placement group-table"
+                belowIds={["status-bar"]}
+              >
+                <TablePagination
+                  id="pagination"
+                  data={sortedRows}
+                  itemName="placement group"
+                  className="u-no-margin--top"
+                  aria-label="Table pagination control"
+                >
+                  <MainTable
+                    id="profile-table"
```

Comment:
```suggestion
                    id="placement-table"
```

Do we even need an id ?